Resources/Prototypes/_Sunrise/Catalog/ert_uplink_catalog.yml

```
@@ -31,7 +31,7 @@
   productEntity: ClothingBackpackDuffelWeaponSubMachineGunP90Filled
   icon: { sprite: _Sunrise/Interface/Misc/nt_icons.rsi, state: p90 }
   cost:
-    Bluecrystal: 17 #C-20r Analog
+    Bluecrystal: 18 #C-20r Analog
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)